meshnets/utils/model_training.py

```diff
@@ -60,6 +60,9 @@ def train_model(config, experiment_config, train_dataset, validation_dataset):
     # Checkpoint config
     save_top_k = experiment_config['save_top_k']
 
+    # Compute the training dataset stats for normalization
+    train_stats = train_dataset.dataset[train_dataset.indices].get_stats()
```

Comment:
Why doesn't `train_dataset.get_stats()` work?

Reply:
I think the Subset class of torch does not offer access to their hyperclass methods such as .get_stats(), but I will check that.